src/Appwrite/Utopia/Response/Model/AttributeList.php

```diff
@@ -11,9 +11,9 @@ class AttributeList extends Model
     public function __construct()
     {
         $this
-            ->addRule('sum', [
+            ->addRule('total', [
                 'type' => self::TYPE_INTEGER,
-                'description' => 'Total sum of items in the list.',
+                'description' => 'Total amount of attributes in a collection.',
```

Comment:
```suggestion
                'description' => 'Total amount of attributes in the given collection.',
```

Reply:
Maybe this will make it more consistent with memberships: `Total number of collection attributes.`

Reply:
Yes, please also use `total number` instead of `total amount`. In English, we use "the number of" for countable things, and "the amount of" for non-discrete stuff like weight or volume. It's a weird grammar thing.

This is one of those weird things that I still mess up all the time.